sftp/1.3/modules/ROOT/pages/sftp-list.adoc

```diff
@@ -1,15 +1,34 @@
 = List Files Using the SFTP Connector - Mule 4
 :page-aliases: connectors::sftp/sftp-list.adoc
 
-The `list` operation returns a list of messages, in which each message represents any file or folder found within the directory path (`directoryPath`).
+Anypoint Connector for SFTP (SFTP Connector) List operation returns a list of messages, in which each message represents any file or folder found within the directory path:
 
-By default, the operation does not read or list files or folders within any subfolders of `directoryPath`.
+* The path you define in the *Directory path* parameter can be absolute, or it can be relative to a working directory.
+* By default, the operation does not read or list files or folders within any subfolders of the directory path.
+* To list files or folders within any subfolders, set the *Recursive* parameter to `TRUE`.
 
-To list files or folders within any subfolders, set the `recursive` parameter to `true`.
+== Configure the Operation in Studio
 
-Set the path in the file configuration by using `<sftp:config/>`. The path can be absolute, or it can be relative to a working directory (`workingDir`).
+To add and configure the List operation in Studio, follow these steps:
 
-The following example lists the contents of a folder and all messages in `directoryPath`, without listing the contents of the subfolders. The `<foreach>` and `<choice>` components handle each directory in the list differently from the way they handle each file.
+. In the *Mule Palette* view, search *SFTP* and select the *List* operation.
+. Drag the *Read* operation onto the Studio canvas.
+. In the *General* tab of the operation configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. Specify the connection information and click *OK*.
+. In the *General* tab, set the *Directory path* field to `~/dropFolder` to set the path of the file to list.
+
+The following screenshot shows the *List* operation configuration:
+
+.List operation configuration
+image::sftp-list-operation.png[List operation configuration in Studio]
+
+In the XML editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<sftp:list doc:name="List" config-ref="SFTP_Config" directoryPath="~/dropFolder"/>
+----
+
+The following XML example lists the contents of a folder and all messages in the directory path, without listing the contents of the subfolders. The For each and Choice components manages each directory in the list differently from the way they manage each file:
```

Comment:
Suggestion: The following XML example lists the folder contents of messages in the directory path without the subfolder contents. 

Choice components manage each directory in the list differently from the way they manage files: